src/graphql/utils/buildGetQueryParams.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Based on code used in graphql-request repo, responsible for building
+ * GET request based GraphQL query requests
```

Comment:
Kinda wonder if we shouldn't just use `graphql-request` 🤔

Reply:
I think we could, I would like to address that as a separate PR and fix all other usages as well, there's not many

Reply:
https://linear.app/rainbow/issue/APP-408/switch-graphql-codegen-to-use-graphql-request-integration-instead-of